documentation/modules/ROOT/pages/hcp-tech-breakdown.adoc

```diff
@@ -77,7 +87,7 @@ In order to do that, cluster service providers can leverage the following node l
 The following rules apply:
 
 * Pods for a Hosted Control Plane tolerate taints for `control-plane` and `cluster`.
-* Pods for a Hosted Control Plane prefer to be scheduled into the same node.
+* Pods for a Hosted Control Plane prefer to be scheduled into the same node. (this mean etcd/api/controller is in the same node)
```

Comment:
By default, all control plane components for a hosted cluster will run in the same node when using non-ha (single replica)  (this can be tuned).